packages/e2e/src/App.tsx

```diff
@@ -32,6 +36,7 @@ const App = () => {
           console.log('onStartPayment');
         }}
         supportedCurrencies={['USDC', 'SOL']}
+        paymentType={'PAYSTREAM' as PaymentRequestType}
```

Comment:
do we want to keep the consol log here ?

Reply:
it's just for example. I think we can leave it there